source/Octopus.Tentacle/Kubernetes/PodLogLineParser.cs

```diff
@@ -62,23 +62,42 @@ static class PodLogLineParser
 
         public static PodLogLineParseResult ParseLine(string line, IPodLogEncryptionProvider encryptionProvider)
         {
-            var logParts = line.Split(new[] { '|' }, 4);
-            if (logParts.Length != 4)
+            var initialParts = line.Split(new[] { '|' }, 2);
+
+            var datePart = initialParts[0];
+
+            var remainingMessage = initialParts[1];
+
+            //get the first 2 control chars and check
+            var encryptionControl = remainingMessage.Substring(0, 2);
```

Comment:
I did play with doing this as a Span, however to do the split later, you have to change to a String anyway (.NET Framework 4.8 doesn't have a Split method for Span), so I chose to ditch it and just keep it simple